DSL/Ruuter.public/POST/templates/permissions/applicant.yml

```diff
@@ -2,7 +2,7 @@ extract_token_data:
   call: http.post
   args:
     # TODO: Use real TIM endpoint
```

Comment:
Note: To be replaced with tim jwt cookie (In Ruuter Guards)

Reply:
Ok